modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryListeningTimeView.kt

```diff
@@ -44,76 +36,58 @@ fun StoryListeningTimeView(
     story: StoryListeningTime,
     modifier: Modifier = Modifier,
 ) {
-    val context = LocalContext.current
-
-    val timeText = StatsHelper.secondsToFriendlyString(story.listeningTimeInSecs, context.resources)
-    val funnyText = FunnyTimeConverter().timeSecsToFunnyText(
-        story.listeningTimeInSecs,
-        context.resources
-    )
-
-    val backgroundColor = story.podcasts[0].toPodcast().getTintColor(false)
-
     Column(
```

Comment:
This isn't related to the changes in this PR, but what about making this Column scrollable? That might help prevent overlap for users who use large font sizes.

<img width="350px" src="https://user-images.githubusercontent.com/4656348/201398866-f1322082-0bf8-4eb9-955a-6ca764205a0f.png" />

Reply:
Column was scrollable actually. I increased cover row height in [66568da](https://github.com/Automattic/pocket-casts-android/pull/561/commits/66568da32662b429637948cf69349540912da17e) to accommodate transformed image height. 

https://user-images.githubusercontent.com/1405144/201461621-63d7a02b-3f60-4de8-9cf1-61b11b150988.mp4